src/database/effects/cards/2-3-050.ts

```
@@ -0,0 +1,70 @@
+
+        // 捨札からインターセプトカードをランダムで手札に加える
+        const interceptCards = stack.processing.owner.trash.filter(
+          card => card.catalog.category === 'intercept'
+        );
+        if (interceptCards.length > 0) {
+          const selectedCards = EffectHelper.random(interceptCards, 1);
+          const selectedCard = selectedCards[0];
+          if (selectedCard) {
+            Effect.move(stack, stack.processing, selectedCard, 'hand');
+          }
+        }
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)